app/components/bali/avatar/index.scss

```diff
@@ -0,0 +1,40 @@
+$icon-size: 2rem;
```

Comment:
```suggestion
$avatar-upload-icon-size: 2rem;
```

Reply:
Maybe think of a better name, `$icon-size` is to generic

Reply:
I don't think that this is necessary because this variable is inside of Avatar::Component scss file instead of variables.scss. I think that it is obvious that it refers to the avatar icon